backport of https://github.com/root-project/root/pull/2205 to ROOT 6.14.06
to fix 'make install' problem with missing 'vdt/vdtMath.h'
backported by Kenneth Hoste (HPC-UGent)
--- root-6.14.06/cmake/modules/SearchInstalledSoftware.cmake.orig	2019-01-11 16:31:25.306240338 +0100
+++ root-6.14.06/cmake/modules/SearchInstalledSoftware.cmake	2019-01-11 16:32:04.726037833 +0100
@@ -1457,6 +1457,7 @@
         message(STATUS "VDT not found. Ensure that the installation of VDT is in the CMAKE_PREFIX_PATH")
         message(STATUS "               Switching ON 'builtin_vdt' option")
         set(builtin_vdt ON CACHE BOOL "Enabled because external vdt not found (${vdt_description})" FORCE)
+        set_property(GLOBAL APPEND PROPERTY ROOT_BUILTIN_TARGETS VDT)
       endif()
     endif()
   endif()
